Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -122,6 +136,16 @@ public struct PostgresBindings: Sendable, Hashable {
         init<Value: PostgresEncodable>(value: Value) {
             self.init(dataType: Value.psqlType, format: Value.psqlFormat)
         }
+        
+        @usableFromInline
+        var description: String {
+            "\(self.dataType)"
+        }
+        
+        @usableFromInline
+        var debugDescription: String {
+            "\(self.format == .text ? "text" : "bin")(\(self.dataType))"
+        }
```

Comment:
I don't think we should add this conformance here. Instead we should unwrap directly in `PostgresBindings`